source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1416,27 +1429,25 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
     // Processing code uses this specific error code in the case that a
     // message was received out of order.
     stats_.spurious_msgs_received_.inc();
-    // When a message is received out of order, ignore it and also
-    // ignore the stream for the rest of this filter instance's lifetime
-    // to protect us from a malformed server.
```

Comment:
Should we keep this?

Reply:
This comment is kept under if (config_->failureModeAllow())  as below